lemminx-maven/pom.xml

```diff
@@ -131,34 +138,22 @@
 			<scope>compile</scope>
 		</dependency>
 		<dependency>
-			<groupId>io.takari.aether</groupId>
-			<artifactId>aether-connector-okhttp</artifactId>
-			<version>0.17.8</version>
-                        <!-- Parts of these got merged into maven-resolver-api and would cause conflicts if on the classpath
-                             See https://github.com/eclipse/lemminx-maven/issues/360
-                        -->
-			<exclusions>
-				<exclusion>
-					<groupId>org.eclipse.aether</groupId>
-					<artifactId>aether-api</artifactId>
-				</exclusion>
-				<exclusion>
-					<groupId>org.eclipse.aether</groupId>
-					<artifactId>aether-util</artifactId>
-				</exclusion>
-			</exclusions>
+			<groupId>org.apache.maven.resolver</groupId>
+			<artifactId>maven-resolver-transport-file</artifactId>
+			<version>${maven-resolver.version}</version>
 		</dependency>
 		<dependency>
-			<groupId>org.apache.maven.wagon</groupId>
-			<artifactId>wagon-http</artifactId>
-			<version>3.5.3</version>
+			<groupId>org.apache.maven.resolver</groupId>
+			<artifactId>maven-resolver-transport-http</artifactId>
+			<version>${maven-resolver.version}</version>
 		</dependency>
 		<dependency>
-			<groupId>org.apache.maven.wagon</groupId>
-			<artifactId>wagon-file</artifactId>
-			<version>3.5.3</version>
+			<groupId>org.apache.maven.resolver</groupId>
+			<artifactId>maven-resolver-transport-wagon</artifactId>

```

Comment:
Are all of the new dependencies necessary?
Only from looking at the code I see no matching imports.

Reply:
Maven uses tons of dependency injection and most likely reference things with
```java
@Component(hint="wagon")
Transport wagonTransport;
```
or even more dynamic
```java
mavenContext.getComponent(Transport.class, uri.getScheme()); // resolves to FileTransport, or HttpTransport or whatever has support in the classpath
```
which do not surface in imports but are necessary for proper execution.

Reply:
Those are leftovers in our case as we actually use the only `maven-resolver-transport-http`  - removed 
two others